manifest/manifest.json

```diff
@@ -1,10 +0,0 @@
-{
-	"name": "Northstar.DiscordRPC",
-	"displayname": "DPRESENCE",
-	"dependencyName": "DISCORDRPC",
-	"description": "Discord Rich Presence for Northstar",
-	"api_version": "3",
-	"version": "3.0",
```

Comment:
With `manifest.json` being gone, we should probably set version number in `Cargo.toml` correctly to v4 ^^

https://github.com/R2Northstar/NorthstarDiscordRPC/pull/17/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R3

Reply:
done